packages/framework/fluid-framework/README.md

```diff
@@ -1,11 +1,11 @@
 # fluid-framework
 
-The `fluid-framework` package bundles a collection of Fluid Framework client libraries for easy use when paired with a corresponding service client package (e.g. `@fluidframework/azure-client`, `@fluidframework/tinylicious-client`, or `@fluid-experimental/osdp-client (BETA)`).
+The `fluid-framework` package bundles a collection of Fluid Framework client libraries for easy use when paired with a corresponding service client package (e.g. `@fluidframework/azure-client`, `@fluidframework/tinylicious-client`, or `@fluidframework/odsp-client`).
```

Comment:
It is still marked `@beta`, though not experimental.  Maybe keep the (BETA) marking?

Reply:
Sure, will restore it